CODEOWNERS

```diff
@@ -22,3 +22,4 @@
 /OracleWebCenterSites/    @prshshuk
 /OracleWebLogic/          @mriccell
 /OracleSOASuite/          @rdas0405
+/OracleIdentityGovernance/  @rishiagarwal-oracle
```

Comment:
Please put this line in the right place in this file so it remains alphabetical.

Reply:
done.